ci/scripts/create-and-update-db.sh

```diff
@@ -33,11 +33,18 @@ if ! contains "$DATABASES" "$db_user"; then
   DATABASES="$db_user $DATABASES"
 fi
 
+
+
 for db in ${DATABASES}; do
 
   # Create database
-  psql_adm -d postgres -l | awk '{print $1}' | grep -q "${db}" || \
-    psql_adm -d postgres -c "CREATE DATABASE ${db} OWNER ${db_user}"
+  if ! [ "${db}" = "uaadb" ]; then
```

Comment:
This would run for everything that isn't `uaadb` but that means it could run for other `postgres` databases besides CredHub right? I don't know if this is the best solution. I think we should just target `credhub` as a the database and do the specific stuff that we need to do there.